assets/bib/packages-r.bib

```diff
@@ -182,6 +182,13 @@ @manual{highfrequency
 	url          = {https://CRAN.R-project.org/package=highfrequency},
 	note         = {R package version 0.9.5},
 }
+@manual{httr2,
+  title        = {httr2: Perform HTTP Requests and Process the Responses},
+  author       = {Hadley Wickham},
+  year         = {2024},
+  note         = {R package version 1.0.3, https://github.com/r-lib/httr2},
+  url          = {https://httr2.r-lib.org},
+}
```

Comment:
We should also cite tidyfinance

Reply:
Our {tidyfinance} citation is the Tidy Finance with R book. Seems pointless to me to cite the book in the book.